glusterd2/transaction/transaction.go

```diff
@@ -101,7 +101,8 @@ func (t *Txn) checkAlive() error {
 
 	for _, node := range t.Nodes {
 		// TODO: Using prefixed query, get all alive nodes in a single etcd query
-		if !store.Store.IsNodeAlive(node) {
+		_, online := store.Store.IsNodeAlive(node)
```

Comment:
Suggestion: You can use the short notation as that variable isn't used elsewhere 

```go
if _, online := store.Store.IsNodeAlive(p.ID); !online {
    ...
}
```